meta-lmp-base/recipes-bsp/efitools/efitools/0001-lockdown-systemd-boot-conf.patch

```diff
@@ -0,0 +1,23 @@
+From aed83c3dc4fa31db803e212ccc5621aa6aabfe63 Mon Sep 17 00:00:00 2001
+From: Jorge Ramirez-Ortiz <jorge@foundries.io>
+Date: Tue, 27 Aug 2024 11:45:42 +0200
+Subject: [PATCH] lockdown: systemd-boot conf
+
+Signed-off-by: Jorge Ramirez-Ortiz <jorge@foundries.io>
+---
+ lockdown.conf | 2 ++
+ 1 file changed, 2 insertions(+)
+ create mode 100755 lockdown.conf
+
+diff --git a/lockdown.conf b/lockdown.conf
+new file mode 100755
+index 0000000..4d7a943
+--- /dev/null
++++ b/lockdown.conf
+@@ -0,0 +1,2 @@
++title UEFI Secure Boot Provisioning
++efi /LockDown.efi
+\ No newline at end of file
+-- 
+2.34.1
```

Comment:
You don't need a patch for this, you can add the file directly at SRC_URI.

Reply:
sure, I thought the patch will be better, anyway, let me update